src/Directory.Build.props

```diff
@@ -11,12 +11,16 @@
   </PropertyGroup>
 
   <!-- Test runner configuration -->
+  <ItemGroup>
+    <None Include="$(MSBuildThisFileDirectory)Common\Test\App.config" CopyToOutputDirectory="PreserveNewest" Condition="'$(IsUnitTestProject)' == 'true'" />
+  </ItemGroup>
+
+  <!-- Code analyzers -->
```

Comment:
The various analyzers were put under a "Test runner configuration" section so I moved them in a separate group.